winui/AutoComplete/No-Result-Found.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` property and  the desire text to be displayed for indicating no results found can be customized using the `NoResultsFoundTemplate` property. 
+
+## NoResultsFoundContent 
+
+We can customize the Displaying text by using the `NoResultsFoundContent` property. The below code shows how to include the `NoResultsFoundContent` in AutoComplete.
+
+{% tabs %}
+{% highlight xaml %}
+
+ <editors:SfAutoComplete  Header="AutoComplete with No Result Found Text" 
+                          PlaceholderText="Search a country" 
+                          DisplayMemberPath="Country" 
+                          TextMemberPath="Country"
+                          Width="300"   
+                          NoResultsFoundContent="Not Found"
+                          ItemsSource="{Binding ContinentList }">
+ </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with NoResultsFoundContent](No_Results_Found_images\No-Results-Found-Content.png)
+
+## NoResultsFoundTemplate 
+
+We can customize the appearance of the displaying text by using the `NoResultsFoundTemplate` property. The below code shows how to include the `NoResultsFoundTemplate` in AutoComplete.

```

Comment:
@YuvarajMasilamani, change content as

We can customize the appearance of the desire text to be displayed for indicating no results found by using the `NoResultsFoundTemplate` property. The below code shows how to include the `NoResultsFoundTemplate` in AutoComplete.

Reply:
Updated